nixos/modules/programs/weylus.nix

```diff
@@ -0,0 +1,46 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.weylus;
+in
+{
+  options.programs.weylus = with types; {
+    enable = mkEnableOption "weylus";
+
+    openPorts = mkOption {
```

Comment:
@legendofmiracles - in NixOS we call this option `openFirewall`. Aside from that, this module LGTM :+1:

Reply:
@legendofmiracles please address this comment.

Reply:
I don't know how I overlooked this for 2 days straight 😅, thanks for the review.